htdocs/fichinter/agenda.php

```diff
@@ -71,15 +71,15 @@
 // Initialize technical objects
 $object = new Fichinter($db);
 $extrafields = new ExtraFields($db);
-$diroutputmassaction = $conf->fichinter->multidir_output[$conf->entity].'/temp/massgeneration/'.$user->id;
+$diroutputmassaction = $conf->ficheinter->multidir_output[$conf->entity].'/temp/massgeneration/'.$user->id;
```

Comment:
With v18+ you can now use
`$diroutputmassaction = getMultidirOutput($object).'/temp/massgeneration/'.$user->id;`